### ISSUE TITLE
[Accessibility]: Branch/Tag/Commit: Focused item text has very poor color contrast

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35473]
### Audit type

General/Other

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34417

### Problem description

<img width="646" alt="image" src="https://user-images.githubusercontent.com/9516420/168595733-6324e815-375c-4c3f-aaeb-7d0f7d969800.png">

Focused item text is white, which makes it difficult to read.

### Expected behavior

The text is readable and passes color contrast checks

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
Cannot reproduce
